docs/src/fpga.md

```diff
@@ -70,6 +70,20 @@ $ cargo xtask-fpga fpga build-test --target-host $SSH-FPGA-NAME # Build test bin
 $ cargo xtask-fpga fpga test --target-host $SSH-FPGA-NAME # Run test suite on FPGA.
 ```
 
+A developer that only wishes to run a single test on the FPGA can replace the last command with the following:
```

Comment:
nit: Maybe use subsections to split the two sequences? Makes it easier to skim IMO.

Reply:
Done. LMK what you think.